### PR TITLE
RimSort: add 1.0.9

### LINKED ISSUE #1196: [Request]: RimSort
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used
- [X] A game or a game-related tool
- [X] English interface (or at least English documentation)
- [X] Latest stable or development version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

RimSort

### Description

RimSort is an open source mod manager for the video game RimWorld.

### Homepage

https://github.com/RimSort/RimSort

### Download Link(s)

https://github.com/RimSort/RimSort/releases

### Some Indication of Popularity/Repute

Project is very active, with 220+ stars, several contributors, and a Discord server.
It's a community-managed and FOSS alternative to [RimPy Mod Manager](https://github.com/rimpy-custom/RimPy/releases) (which has its own [Scoop package](https://github.com/Calinou/scoop-games/blob/master/bucket/rimpy.json))

### PR DESCRIPTION
**Pull request by Big-Iron-Cheems**
Closes #1196

- [x] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Big-Iron-Cheems** [comment]
I think this should fix it, but I may be off track, first time contribution.

File: bucket/rimsort.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/RimSort/RimSort/releases/download/v1.0.9/RimSort-v1.0.9-Windows_x86_64.zip",
+            "hash": "ccf8623044dc1e81643f8ec908253cf4c3e75d262558adf6c1d15de410faf8e5"
+        },
+        "32bit": {
+            "url": "https://github.com/RimSort/RimSort/releases/download/v1.0.9/RimSort-v1.0.9-Windows_x86_64.zip",
+            "hash": "ccf8623044dc1e81643f8ec908253cf4c3e75d262558adf6c1d15de410faf8e5"
+        },
```
Comment (Calinou) [inline bucket/rimsort.json:14]:
```suggestion
```

Comment (Calinou) [inline bucket/rimsort.json:14]:
```suggestion
        }
```

File: bucket/rimsort.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        "github": "https://github.com/RimSort/RimSort"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/RimSort/RimSort/releases/download/v$version/RimSort-v$version-Windows_x86_64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/RimSort/RimSort/releases/download/v$version/RimSort-v$version-Windows_x86_64.zip"
+            }
```
Comment (Calinou) [inline bucket/rimsort.json:35]:
```suggestion
            }
```

**Comment by Big-Iron-Cheems** [comment]
Is this ready to be merged?

**Comment by Calinou** [comment]
> Is this ready to be merged?

I was waiting to see if @github-actions would post its comment again after this PR had new commits, but it didn't;

**Pull request merged**